android_sample/jni/amber_script.cc

```diff
@@ -20,10 +20,10 @@ namespace amber {
 namespace android {
 namespace {
 
-const std::string kAmberDir("amber/");
-const std::string kAmberScriptExtension(".amber");
-const std::string kShaderNameSignature(".vk_shader_");
-const std::string kShaderExtension(".spv");
+const char* kAmberDir = "amber/";
```

Comment:
const char kAmberDir[] = "amber/";

(This was the way the security person said these should be done, I don't know why, heh)